frr-tools/cp-tool/cp-tool.go

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func CopyFile(srcFile string, destFile string) error {
+	srcFile = filepath.Clean(srcFile)
+	destFile = filepath.Clean(destFile)
```

Comment:
nit: rename the input to src,dst (so the new vars are declared with new names)

Reply:
renamed the inputs